fdbcli/DebugCommands.actor.cpp

```diff
@@ -100,9 +102,9 @@ ACTOR Future<bool> getKeyServers(
 // The command is used to get all storage server addresses for a given key.
 ACTOR Future<bool> getLocationCommandActor(Database cx, std::vector<StringRef> tokens) {
 	if (tokens.size() != 2 && tokens.size() != 3) {
-		printf("getlocation <KEY> [<KEY2>]\n"
-		       "fetch the storage server address for a given key or range.\n"
-		       "Displays the addresses of storage servers, or `not found' if location is not found.");
+		fmt::println("getlocation <KEY> [<KEY2>]\n"
+		             "fetch the storage server address for a given key or range.\n"
+		             "Displays the addresses of storage servers, or `not found' if location is not found.");
```

Comment:
The last line here wasn't getting printed due to missing `\n` at the end -- until the user types something or another output, hence changed this to println.